src/main.js

```diff
@@ -119,21 +119,21 @@ $(document).ready(() => {
       }
 
       if (reload) {
-        tryLoadRepo(true);
+        await tryLoadRepo(true);
       }
     }
 
-    function tryLoadRepo(reload) {
-      const token = octotree.getAccessToken();
-      adapter.getRepoFromPath(currRepo, token, (err, repo) => {
+    async function tryLoadRepo(reload) {
+      const token = await octotree.getAccessToken();
+      await adapter.getRepoFromPath(currRepo, token, async (err, repo) => {
```

Comment:
No need `await` for `getRepoFromPath`, the next step after calling `getRepoFromPath` is on callback.

Reply:
There are places calling tryLoadRepo, so either need to return the adapter.getRepoFromPath or await for it